From 3af069ea2b04bb5d99d908951dddf0da0b25b6ba Mon Sep 17 00:00:00 2001
From: Davide Ciminaghi <ciminaghi@gnudd.com>
Date: Tue, 25 Mar 2014 18:10:17 +0100
Subject: [PATCH 26/83] mips, mach-ath79/gpio.h: increase ARCH_NR_GPIOS to 512
 This is needed by gpio-mcuio (which registers additional gpios starting from
 100).

Signed-off-by: Davide Ciminaghi <davide@linino.org>
---
 arch/mips/include/asm/mach-ath79/gpio.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/mips/include/asm/mach-ath79/gpio.h b/arch/mips/include/asm/mach-ath79/gpio.h
index 60dcb62..fdb8fa8 100644
--- a/arch/mips/include/asm/mach-ath79/gpio.h
+++ b/arch/mips/include/asm/mach-ath79/gpio.h
@@ -13,7 +13,7 @@
 #ifndef __ASM_MACH_ATH79_GPIO_H
 #define __ASM_MACH_ATH79_GPIO_H
 
-#define ARCH_NR_GPIOS	64
+#define ARCH_NR_GPIOS	512
 #include <asm-generic/gpio.h>
 
 int gpio_to_irq(unsigned gpio);
-- 
2.10.1

